subscription/remove-seats.md

```diff
@@ -6,26 +6,27 @@ redirect_from:
 - /docker-hub/billing/remove-seats/
 ---
 
-You can manage your Docker Hub subscription anytime by removing seats from your subscription. If you remove seats in the middle of the billing cycle, changes are applied immediately and reflect in the next billing cycle.
+You can remove seats from your Team or Business subscription at anytime. 
+
+If you remove seats in the middle of the billing cycle, changes are applied immediately and are reflected in the next billing cycle.
```

Comment:
**suggestion:** may be worth mentioning that unused period of removed seats isn't refundable nor creditable.